### PR TITLE
Load models directly to GPU when voodooray is disabled

### PR DESCRIPTION
**Pull request by ninele7**
When testing xformers on colab I was running OOM (probably because they use some additional ram for init). 

I changed loading behavior when voodooray is disabled, to load model directly into vram, instead of passing it through CPU ram. It fixed problem on colab for me.

Also added channels_last memory format when not using voodooray (it is used with voodooray) https://pytorch.org/tutorials/intermediate/memory_format_tutorial.html.

TLDR: lower ram usage on init when voodooray is disabled.

### CODE REVIEW

**Comment by ninele7** [comment]
@db0 Linter fails on `bridge.py`, that I haven't touched.

**Comment by db0** [comment]
@hlky  it's your baby. Looks ok to you?

File: nataili/model_manager.py [hunk 1]
```diff
@@ -190,14 +189,17 @@ def load_model_from_config(self, model_path="", config_path="", map_location="cp
...
-        model = model if precision == "full" else model.half()
         if not self.disable_voodoo:
+            model = self.load_model_from_config(model_path=ckpt_path, config_path=config_path)
+            model = model if precision == "full" else model.half()
             logger.debug(f"Doing voodoo on {model_name}")
             model = push_model_to_plasma(model) if isinstance(model, torch.nn.Module) else model
         else:
-            model = (model if precision == "full" else model.half()).to(device)
+            model = self.load_model_from_config(
+                model_path=ckpt_path, config_path=config_path, map_location=f"cuda:{gpu_id}"
```
Comment (hlky) [inline nataili/model_manager.py:200]: just to note cuda map location does use more vram, and in some cases would be slower than cpu

**Review by hlky — APPROVED** [review]

File: nataili/model_manager.py (hunk 1 above)
Comment (ninele7) [reply nataili/model_manager.py:200]: When loading I saw serious memory usage increase (more than 6GB was used to start). During generation I saw only 60Mb. Is it critical?

**Comment by ninele7** [comment]
@db0 So, with approve from hlky, will you merge it? Or I need to add ability to make it optional?

**Comment by db0** [comment]
If it's good enough for @hlky , it's good enough for me :) We can re-adjust if we notice any problems
